web/src/components/tasks/anytime-action/tax-clearance-certificate/AnytimeActionTaxClearanceCertificateElement.tsx

```diff
@@ -41,8 +41,8 @@ export const AnytimeActionTaxClearanceCertificateElement = (props: Props): React
   );
   const [formationAddressData, setAddressData] = useState<FormationAddress>(emptyFormationAddressData);
   const [profileData, setProfileData] = useState<ProfileData>(createEmptyProfileData());
-  const [certificatePdfArray, setCertificatePdfArray] = useState<number[] | undefined>(
-    props.CMS_ONLY_certificatePdfArray || undefined
+  const [certificatePdfBlob, setCertificatePdfBlob] = useState<Blob | undefined>(
```

Comment:
[dust] Curious as to the choice of `Blob` versus `Uint8Array` or `ArrayBuffer`. I'd probably go with `Blob` myself but is there any particular reason you've chosen it here?